app/javascript/application.js

```diff
@@ -1,3 +1,27 @@
 // Entry point for the build script in your package.json
-import "@hotwired/turbo-rails"
-import "./controllers"
+
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from "react-redux";
+
+import  { store } from './redux/store.js'
+
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { fetchUserData } from './redux/user/userSlice.js';
+import Great from './components/great.js';
+
+
+function App() {
+    const dispatch = useDispatch();
+    useEffect(() => {
+      dispatch(fetchUserData());
+    }, [dispatch]);
+  return (<Great/>);
+}
```

Comment:
Nice job creating an `App` component as required in this milestone👍🏼. To ensure maintainability in your codebase, kindly ensure the `App` component is placed in a separate file and imported when needed👍🏼.


Also, please note that as a requirement, the greeting component should be linked to the app component with the help of react-router😉. Kindly do well to implement this👍🏼.


<img width="803" alt="image" src="https://github.com/francksefu/hello_rails_react/assets/64643995/b43c0406-8a9f-44a5-a0df-3bf4f0aeaf49">